### ISSUE TITLE
Add a control for handling bad student and/or advisor data in cohort file

### ISSUE THREAD

**Issue by mfldavidson** [tl-its-umich-edu/student_explorer#73]
Uploading a file to create a cohort where either the student or the advisor uniqnames do not match an existing student or advisor in the system causes the Advisors page to return a server error. In the below instance this was caused by the upload of a file that BOTH contained headings (studentuniqname, cohortname, and advisoruniqname respectively) and also had a cohort name instead of an advisor uniqname in the second column. The cohort name was "engineering honors" so SE was looking for an advisor with uniqname matching "engineering honors" (or I think "engineering hono" because it got cut off) and couldn't find it so it was returning an error. I will email the file that was used in the upload that broke the system to Zhen. The issue is fixed for now but we are interested in adding a control to handle this problem better in the future.

![image](https://user-images.githubusercontent.com/31591389/52150458-0aa62800-263e-11e9-970c-93c7155e398e.png)

**Comment by jonespm** [comment]
I created #74 which will fix the list to not give this error, but it still gets in there and *could* cause other problems. It couldn't look up the user name because I believe it didn't allow spaces in the username, and this should probably be checked during upload. However I noticed still a few users didn't have first or last name either and they were in there, so I added the username to make them still "work".
